src/book/02-system/04-hardware/01-overview.mdx

```diff
@@ -25,12 +26,41 @@ The following modifications were made to fit our specific requirements:
 - Added a 3d printed brace between the hip yaw servos to reduce flex. The brace also serves as a mount for power switches and the main power fuse.
 - Created custom draws for the computer (intel NUC) and subcontrollers (OpenCR, NUSense)
 - Designed a bearing housing for the hip yaw that takes a tapered roller bearing from a mountain bike head stock. This interfaces between the servo shaft and the hip pitch/roll block and takes the load of the servo shaft and spreads it over a wider area on the bottom of the torso.
+- 3d printed Onyx legs have now been replaced with 'L' angle machined aluminium.
+- The NUC drawer has been replaced with a vertical NUC assembly. These changes were made to utilise space within the torso which now provides extra working space. Features include:
+
+  - Rubber mounts.
+  - Lower mounts that double as a hip yaw motor brace. This addition stiffens the lower torso.
+  - Integrated power assembly.
+  - Subcontroller mount.
+
+- The back panel has been simplified as the old NUC placement was prone to breaking ports on the NUC board. The opportunity for improvements appeared when the new vertical NUC mount placement required cables to access data ports. New back panel features include:
+
+  - Replaceable keystone ports (usb-c, usb-3, and ethernet).
+  - Replaceable data cables to NUC.
+  - New bigger buttons.
+
+<Alert>
+
+The **RED** button relaxes the servos.
+
+The **BLUE** button starts behaviour.
+
+The **BLACK** button stops behaviour.
+
+</Alert>
 
 ## Servos
 
 The arms and head of the NUgus robot use [Dynamixel MX64AR servos](http://emanual.robotis.com/docs/en/dxl/mx/mx-64/), with two in each shoulder, one in each elbow, and two in the neck. The legs use both [Dynamixel XH540-W270-R servo](https://emanual.robotis.com/docs/en/dxl/x/xh540-w270/) and [Dynamixel MX106 servos](http://emanual.robotis.com/docs/en/dxl/mx/mx-106/), with two X series in each ankle, one in each knee, and two in each hip, with the two hip yaw servos remaining as MX-106.
 
-There are custom made mounting brackets milled from aluminium for the [flywheels](https://github.com/NUbots/NUware/blob/main/Igus/Current%20Design/stl_file/Flywheel_Bracket.stl), and servo-to-servo mounting brackets in the [neck](https://github.com/NUbots/NUware/blob/main/Igus/Current%20Design/stl_file/Head_Bracket.stl), hip, and ankle. There are also 3D printed cable clamps on the back of the [knees](https://github.com/NUbots/NUware/blob/main/Igus/Current%20Design/stl_file/Servo_cable_holder.stl), [ankle and hip blocks](https://github.com/NUbots/NUware/blob/main/Igus/Current%20Design/stl_file/Cable_Holder.stl).
+Custom 3d printed Onyx mounts are now used, but over time it has been determined that parts such as the knee servo mounts and the idle wheel mounts may better to be printed from PETG filament.
+
+- The custom mounts that retain MX style idle wheels due to x-series idle horn design are detrimental to the robot and should be upgraded in future.
```

Comment:
```suggestion
- Custom mounts that retain MX style idle wheels on x-series servos. MX style idle wheels are preferred due to being bolted rather than the clip-in design of the x-series idle wheels. 
```